azure-pipelines.yml

```diff
@@ -286,6 +286,25 @@ jobs:
         channels:
           - main
 
+  # Affinitized Scenario benchmarks (Initially just PDN)
+  - template: /eng/performance/build_machine_matrix.yml
+    parameters:
+      jobTemplate: /eng/performance/scenarios.yml
+      buildMachines:
+        - win-x64
+        - win-arm64
+      isPublic: false
+      jobParameters:
+        kind: scenarios
```

Comment:
The `kind` should be something else, isn't it? With this it blends into "regular" scenarios.

Reply:
I was more worried about new affinitized numbers going into old non-affinitized. But as I'm thinking about it, it's probably not a bad thing. We know about it and it's just kind of configuration setting.

Reply:
For this scenario, the name of the affinitized test is also updated to include the affinity in the name to denote the difference.